private-templates-service/client/src/components/Gallery/Gallery.tsx

```diff
@@ -1,21 +1,41 @@
 import * as React from "react";
+import { connect } from 'react-redux';
+import { RootState } from '../../store/rootReducer';
 import AdaptiveCardPanel from "../AdaptiveCardPanel";
 import { Container } from "./styled";
+import { getTemplates } from "../../store/templates/actions";
+
+const mapStateToProps = (state: RootState) => {
+  return { state: state };
+}
```

Comment:
```
const mapStateToProps = (state: RootState) => {
  return {
    templates: state.templates
  }
}
```

Reply:
In every location you use `this.props.state.templates` you can instead use `this.props.templates`